src/ziggurat/messaging/producer.clj

```diff
@@ -151,12 +151,15 @@
 (defn- channel-retry-type [topic-entity channel]
   (:type (channel-retry-config topic-entity channel)))
 
-(defn- get-channel-retry-count [topic-entity channel]
+(defn get-channel-retry-count [topic-entity channel]
   (:count (channel-retry-config topic-entity channel)))
 
+(defn get-configured-retry-count []
```

Comment:
This method should go inside `ziggurat.config`. Missed it while pairing

Reply:
ziggurat.config is giving whole configuration and we are only fetching rabbit-mq `retry count` which is required for our use case.  So i feel this file is the right place for this method as other similar methods are already there.

Reply:
hmm. fine. since channel retry configuration is also coming from producer.clj, it seems fine. But, given that we're just fetching configuration and not doing anything `produce` specific, ziggurat.config would have been the right place even for getting channel retry count.

You can look at other helper methods in `ziggurat.config`

Reply:
Moved config as per suggestion, kindly check and merge.